src/lib/Behat/Page/ObjectStateGroupPage.php

```diff
@@ -111,7 +116,7 @@ public function verifyIsLoaded(): void
             ->setTimeout(3)
             ->waitUntilCondition(new ElementExistsCondition($this->getHTMLPage(), $this->getLocator('objectStatesTable')))
             ->find($this->getLocator('pageTitle'))
-            ->assert()->textEquals(sprintf('Object state group: %s', $this->expectedObjectStateGroupName));
+            ->assert()->textEquals(sprintf($this->expectedObjectStateGroupName));
```

Comment:
```suggestion
            ->assert()->textEquals($this->expectedObjectStateGroupName);
```
`sprintf` call is not needed when we're not combining strings anymore